tmt/steps/report/reportportal.py

```diff
@@ -143,8 +157,77 @@ def go(self) -> None:
             message = response.text
         if not response.ok:
             raise tmt.utils.ReportError(
-                f"Received non-ok status code from ReportPortal, response text is: {message}")
+                "Received non-ok status code from ReportPortal, "
+                f"response text is: {message}")
         else:
             self.debug(f"Response code from the server: {response.status_code}")
             self.debug(f"Message from the server: {message}")
             self.info("report", "Successfully uploaded.", "yellow")
+
+        if self.get("add-meta-data"):
+            uuid = None
+            re_result = re.search("\\w{8}-(\\w{4}-){3}\\w{12}", str(message))
+            if isinstance(re_result, re.Match):
+                uuid = re_result.group()
+
+            if not isinstance(uuid, str):
+                self.debug(f"Message from tmt: could not update metadata in {message}")
```

Comment:
Is it necessary to mention the log message in tmt's logs comes from tmt? I'd expect that to be the default case.

Reply:
You are correct

Reply:
Done